Backend/dev/prometheus/config.yml

```diff
@@ -6,28 +6,29 @@ scrape_configs:
   - job_name: "beckn-app-backend"
     static_configs:
       - targets:
-          - "192.168.0.17:9999"
+          - "10.10.50.232:9999"
```

Comment:
not related change, can you remove this ?

Reply:
Whole file ??
It's somewhere mentioned in `arionProjectConfiguration`